integrations/gcp/gcp_core/utils.py

```diff
@@ -0,0 +1,62 @@
+import base64
+import json
+from typing import TypeVar
+
+import proto  # type: ignore
```

Comment:
Didnt u install the typing package?

Reply:
I did, and I tried 5 more stub libs and still getting mypy errors, for now im removing the typings libs and leaving this